src/pax.rs

```diff
@@ -145,3 +146,74 @@ impl<'entry> PaxExtension<'entry> {
         self.value
     }
 }
+
+/// A builder for PAX extended headers.
+/// It has the data for the extended header
+/// and a status on whether the extended header has been updated.
+pub struct PaxBuilder {
```

Comment:
I don't quite understand what this buys us over just a `Vec<u8>`...the "updated" field is always equivalent to `!data.is_empty()` right?

Reply:
Indeed, the updated on my first version was used for something else that I dropped.
Switching to check if empty